markdown-parsing.js

```diff
@@ -180,6 +183,22 @@ const filters: {[key: string]: (HTMLElement) => string | HTMLElement} = {
   },
   UL(el) {
     return el
+  },
+  DIV(el) {
```

Comment:
Is this something we can do in dotcom by catching the `quote-selection` event? These are very dotcom specific and would be nice to not include in this module.